### PR TITLE
Add resources for tagging and releasing dbt-styleguide

### PR DESCRIPTION
**Pull request by naomiwritescode**
Refer to the documentation pr: fishtown-analytics/knowledge#153

View the tags and releases!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "url": "https://github.com/fishtown-analytics/dbt-styleguide/issues"
+  },
+  "homepage": "https://github.com/fishtown-analytics/dbt-styleguide#readme",
+  "devDependencies": {
+    "auto-changelog": "^2.2.0",
+    "release-it": "^13.6.6"
+  },
+  "release-it": {
+    "github": {
+      "release": false
```
Comment (naomiwritescode) [inline package.json:26]: For automating a github release, the config needs a [personal access token](https://github.com/release-it/release-it/blob/master/docs/github-releases.md). If we want to, we could grant developers these tokens and automate this process as well. Thoughts?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    "npm": {
+      "publish": false
+    },
+    "git": {
+      "addUntrackedFiles": true,
+      "changelog": "npx auto-changelog --stdout --commit-limit false -u --template https://raw.githubusercontent.com/release-it/release-it/master/templates/changelog-compact.hbs",
```
Comment (naomiwritescode) [inline package.json:33]: This is a simple changelog. In the future we can update this to provide more description.

**Comment by jgiuffrida** [comment]
Looks awesome - one thought:

I wonder if we should try to enforce rebase -> squash merges in this repo to keep the release changelog as clean as possible (changelog item -> PR, rather than commit sha)

**Comment by naomiwritescode** [comment]
> I wonder if we should try to enforce rebase -> squash merges in this repo to keep the release changelog as clean as possible (changelog item -> PR, rather than commit sha)

I think this is reasonable. @cmcarthur what do you think?

**Comment by cmcarthur** [comment]
I am ok with rebase/squash if you are. Let me know and I will enable it.

**Review by cmcarthur — APPROVED** [review]

**Pull request merged**